Makefile

```diff
@@ -1,8 +1,8 @@
 CC ?= gcc
 CFLAGS += -std=c99 -Wall -Wextra -pedantic -MMD -ggdb
 
-TARGETS = f3write f3read
-EXTRA_TARGETS = f3probe f3brew f3fix
+TARGETS = f3write f3read f3probe f3brew f3fix
+#EXTRA_TARGETS = f3probe f3brew f3fix
```

Comment:
We cannot make this change because only Linux supports the extra applications, namely, `f3probe`, `f3brew`, and `f3fix`. If we made this change, other platforms would have issues. Please instruct Snap to call `make` **and** `make extra` to build all applications.